crates/shared/src/price_estimation/native_price_cache.rs

```diff
@@ -51,6 +52,7 @@ struct Inner {
     high_priority: Mutex<IndexSet<H160>>,
     estimator: Box<dyn NativePriceEstimating>,
     max_age: Duration,
+    concurrent_requests: usize,
```

Comment:
Since this is now part of `Inner` `concurrent_requests` can be removed from `UpdateTask` and no longer needs to be passed into `estimate_prices_and_update_cache()` (now it's available on `self`).